pkg/bootstrap/config.go

```diff
@@ -327,6 +328,10 @@ func extractRuntimeFlags(cfg *model.NodeMetaProxyConfig) map[string]any {
 		"envoy.reloadable_features.http_reject_path_with_fragment":                                             false,
 		"envoy.restart_features.use_eds_cache_for_ads":                                                         true,
 	}
+	// This flag limits google_grpc to TLSv1.2 as the maximum version.
+	if fips {
+		runtimeFlags["envoy.reloadable_features.google_grpc_disable_tls_13"] = true
```

Comment:
is tls1.3 less secure?

Reply:
No, it's not. But there's no certified TLS1.3 FIPS implementation in Istio.

Reply:
Please add that as a contextual comment here so the reasoning is more obvious for later readers.

Reply:
Ideally with a tracking issue to remove it once 1.3 support is added back in

Reply:
That would be a net new feature.  TLS 1.3 was never supported in FIPS mode, and we don't really control NIST timelines.